platform/linux-generic/m4/configure.m4

```diff
@@ -5,30 +5,33 @@ ODP_ATOMIC
 
 ODP_PTHREAD
 ODP_TIMER
-AC_ARG_WITH([openssl],
-	    [AS_HELP_STRING([--without-openssl],
-			    [compile without OpenSSL (results in disabled crypto and random support)])],
-	    [],
-	    [with_openssl=yes])
-AS_IF([test "$with_openssl" != "no"],
-      [ODP_OPENSSL])
-AM_CONDITIONAL([WITH_OPENSSL], [test x$with_openssl != xno])
+m4_include([platform/linux-generic/m4/odp_pcap.m4])
+m4_include([platform/linux-generic/m4/odp_scheduler.m4])
 
 AC_ARG_WITH([pcap],
 	    [AS_HELP_STRING([--without-pcap],
-			    [compile without PCAP])],
+			    [(linux-generic) compile without PCAP])],
```

Comment:
I'd prefer moving the linux-generic text after the description with square brackets to distinguish it from default value.

Reply:
In that case shouldn't we change the default value texts to follow autoconf built-in options? I.e. running `./configure --help` gives me:
```
...
  --enable-shared[=PKGS]  build shared libraries [default=yes]
  --enable-static[=PKGS]  build static libraries [default=yes]
  --enable-fast-install[=PKGS]
                          optimize for fast installation [default=yes]
...

```

Reply:
Yes, you'r correct. So something like:
`--enable-scheduler-default  choose default scheduler [default=basic] (linux-generic)`

Reply:
Right. I'm going through the current `AC_ARG_*` and re-formatting them.